### ISSUE TITLE
Markings not showing up on Thunderbird project inside cloud9

### ISSUE THREAD

**Issue by arshdkhn1** [michaelmitchell/c9.ide.language.javascript.eslintd#4]
Hi, I was trying to use eslint for Thunderbird project. I followed all the steps given on readme.md but unfortunately i dont see any markings. I would also like to have these marking on .jsm files. Could please help me configure that out ?

**Comment by michaelmitchell** [comment]
Can you confirm you added your own .eslintrc  to your project? The plugin will not work if no .eslintrc is found, I am unable to advise you how to make eslint work with .jsm files, you will have to check the eslint documentation to see if that is possible.

**Comment by arshdkhn1** [comment]
@michaelmitchell yes there are .eslintrc files in the project. You can see it yourself [here](https://dxr.mozilla.org/comm-central/).

**Comment by michaelmitchell** [comment]
If you give me some specific files you have tested on I will take a look when I have some time.
